### PR TITLE
[MODFISTO-477] - A new encumbrance is needed during FYRO for orders with "Re-encumber" = false

### PR DESCRIPTION
**Pull request by Abdulkhakimov**
## Purpose
_Describe the purpose of the pull request._

## Approach
_How does this change fulfill the purpose?_

### TODOS and Open Questions
- [ ] Use GitHub checklists. When solved, check the box and explain the answer.

### Learning
[MODFISTO-477](https://folio-org.atlassian.net/browse/MODFISTO-477)

### CODE REVIEW

**Comment by Abdulkhakimov** [comment]
image

File: acquisitions/src/main/resources/thunderjet/cross-modules/features/remove_fund_distribution_after_rollover_when_re-encumber_false.feature [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+    Then status 204
+
+    Examples:
+      | fiscalYearId     | code     |
+      | fromFiscalYearId | fromYear |
+      | toFiscalYearId   | toYear   |
+
+
+  Scenario: Remove fund distribution from POL
+    * configure headers = headersUser
```
Comment (damien-git) [inline acquisitions/src/main/resources/thunderjet/cross-modules/features/remove_fund_distribution_after_rollover_when_re-encumber_false.feature:198]: This is not needed here (headersUser is the default).

File: acquisitions/src/main/resources/thunderjet/cross-modules/cross-modules.feature [hunk 2]
```diff
@@ -153,6 +153,9 @@ Feature: cross-module integration tests
   Scenario: Pending payment update after encumbrance deletion
     Given call read('features/pending-payment-update-after-encumbrance-deletion.feature')
 
+  Scenario: Remove fund distribution after rollover from open order with re-encumber flag is false
+    Given call read('features/remove_fund_distribution_after_rollover_when_re-encumber_false.feature')
```
Comment (damien-git) [inline acquisitions/src/main/resources/thunderjet/cross-modules/cross-modules.feature:157]: Feature files are usually using hyphens in files names, and we should keep it consistent.

**Review by damien-git — CHANGES_REQUESTED** [review]

File: acquisitions/src/main/resources/thunderjet/cross-modules/features/remove_fund_distribution_after_rollover_when_re-encumber_false.feature (hunk 1 above)
Comment (Abdulkhakimov) [reply acquisitions/src/main/resources/thunderjet/cross-modules/features/remove_fund_distribution_after_rollover_when_re-encumber_false.feature]: Thanks, removed

File: acquisitions/src/main/resources/thunderjet/cross-modules/cross-modules.feature (hunk 2 above)
Comment (Abdulkhakimov) [reply acquisitions/src/main/resources/thunderjet/cross-modules/cross-modules.feature]: Thank you. Renamed

Comment (damien-git) [reply acquisitions/src/main/resources/thunderjet/cross-modules/cross-modules.feature]: Sorry, I meant you should replace the underscores with hyphens.

Comment (azizbekxm) [reply acquisitions/src/main/resources/thunderjet/cross-modules/cross-modules.feature]: remove-fund-distribution-after-rollover-when-re-encumber-false

Comment (Abdulkhakimov) [reply acquisitions/src/main/resources/thunderjet/cross-modules/cross-modules.feature]: Done

**Review by azizbekxm — APPROVED** [review]

**Review by SerhiiNosko — APPROVED** [review]

**Review by damien-git — APPROVED** [review]

**Pull request merged**